src/components/Home.vue

```diff
@@ -76,11 +76,9 @@ export default {
     noop() {}
   },
   mounted() {
-    const invalidTokenRegex = /Invalid token/
     this.$kuzzle.on('tokenExpired', () => this.onTokenExpired())
     this.$kuzzle.on('queryError', e => {
-      // TODO use the error codes when available
-      if (invalidTokenRegex.test(e.message)) {
+      if (e.code === 401) {
```

Comment:
Ok, my bad... I meant we should use the `id` attribute since it's more human-readable.